man/src/network/create.md

```diff
@@ -134,3 +134,44 @@ $ docker network create -d overlay \
   --opt encrypted=true \
   my-ingress-network
 ```
+
+### Run services on predefined networks
+
+You can now create services on the predefined docker networks `bridge` and `host`.
+
+```bash
+$ docker service create --name my-service \
+  --network host \
+  --replicas 2 \
+  busybox top
+```
+
+### Swarm networks with local scope drivers
+
+You can now create a swarm network with local scope network drivers. You do so
+by promoting the network scope to `swarm` during the creation of the network. 
+You will then be able to use this network when creating services. 
+
+```bash
+$ docker network create -d bridge \
+  --scope swarm \
+  --attachable \
+  swarm-network
+```
+
+For network drivers which provide connectivity across hosts (ex. macvlan), if
+node specific configurations are needed in orther to plumb the network on each
+host, you will supply that configuration via a configuration only network.
+When you create the swarm scoped network, you will then specify the name of the 
+network which contains the configuration.
+
+
+```bash
+node1$ docker network create --config-only --subnet 192.168.100.0/24 --gateway 192.168.100.115 mv-config
```

Comment:
@thaJeztah 
> For overlay networks, basically the same is done currently, only the "config" (/"defaults") can be automatically determined by swarm/libnetwork, so all this takes place behind the scenes

Just a clarification. For swarm overlay networks, libnetwork does not take care of any config behind the scene. The ovl nw resource allocation is centrally done or retrieved by the swarm allocator. Same for any nw config user may specify (`subnet`).

This is possible because of the nature of the overlay driver network: it is an overlay on existing underlay network infrastructure. It does not need to know the node's underlay networking.
 
> question: I assume the network needs to have the same ID on each node

Yes, this is the fundamental requirement, to guarantee the service discovery across the cluster.
The swarm multi-host network needs to be created by, and its life cycle controlled by the swarm leader, as @mavenugo explained.

> docker network create --config can be run on any node. So far we had a "least privilege" model in swarm; do we cross that boundary here?

I don't think we do, because a config-only network is local only and cannot be used to run containers or services.